src/bulk-entity-replacer.py

```diff
@@ -87,33 +100,48 @@ def fixEnt(ent, b):
 
     args = cmdParser.positionalArguments()
     configArg = args[0]
+    configPath = Path(configArg).absolute().resolve()
+    if not configArg or not configPath.is_file():
+        print("Invalid config path!")
+        sys.exit()
+
+    workingDirectory = configPath.parent
 
     stbArg = cmdParser.isSet(stbOpt)
 
-    with open(configArg) as configFile:
+    with open(configPath) as configFile:
         config = json.load(configFile)
 
+        if "stb" in config and config["stb"]:
+            stbArg = True
+
         totalRooms = 0
-        print("Replacing entities:", config["entities"])
-        for rf in config["files"]:
-            print("File: ", rf)
-            if ".xml" not in rf:
-                print("Must be xml! Skipping!")
-                continue
-
-            roomFile = cvt.xmlToCommon(rf)
+        printf("Replacing entities:", config["entities"])
+
+        files = []
+        for file in config["files"]:
+            basePath = workingDirectory / Path(file)
+            if not basePath.is_dir() and basePath.suffix != ".xml":
```

Comment:
I like how you split the loop; this is more readable! nit: Reversing the branches reads better to me
```suggestion
            if basePath.is_dir() or basePath.suffix is ".xml":
```